### ISSUE TITLE
Invariant with version 7.4.3

### ISSUE THREAD

**Issue by frontendphil** [react-dnd/react-dnd#1308]
**Describe the bug**
While trying to update to the latest patch version we started seeing

```
Invariant Violation: In order to access an instance of the decorated component, it must either be a class component or use React.forwardRef()
```

in our specs.

We are not using the new hooks API. What further info would you need? We are using a `DragSource` and a `DropTarget` in the same component.

I do suspect that this change: https://github.com/react-dnd/react-dnd/commit/6440c548a72f4076d4c9de4b9e91d14fdf99caf4 has something to do with it. 

I, also, do suspect this is somewhat related to #1307 since only the specs in which we're testing sort functionality are failing.

**Comment by darthtrevino** [comment]
Are you using a function component?

**Comment by darthtrevino** [comment]
This should be fixed with v7.4.4

**Comment by frontendphil** [comment]
Yes, we're using a function component. 7.4.4 seems to have fixed the issue. Pipeline is green again 👍
